faithcomesbyhearing/dbp
# FCBHDBP-175 Evaluate whether DBP requires package.json / npm. Pare down or delete

Evaluate whether DBP requires package.json / npm. Pare down or delete



# Description
- Supported to use laravel-mix as a dev tool for the frontend.
- It has updated the laravel-mix to the lastest version.
- Kept the Vue library to use the AlgoliaBibleSearch component.
- Moved the app_api_key.scss file to the resources/assets/sass/ folder.
- Updated the mix scripts into the package.json.
- Updated the file: resources/assets/js/app.js to keep the used dependencies.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-175

## How Do I QA This
- Execute `npm install`
- Execute `npm run watch`
- You should see the watcher to reload the browser if the source files have changes how you can see in the next videos:

### Demo 1

https://user-images.githubusercontent.com/73488660/133362964-ce119678-411e-4ae5-abea-b77a2348f478.mp4

### Demo 2

https://user-images.githubusercontent.com/73488660/133362974-7e2f52e8-108a-4cea-8272-ef37edb221a8.mp4



## Views changed

### /login
![screenshot-localhost_3000-2021 09 14-21_17_01](https://user-images.githubusercontent.com/73488660/133363084-8b2a6dfe-b127-4f90-90a8-ec46c9c19ba9.png)

### /register
![screenshot-localhost_3000-2021 09 14-21_17_28](https://user-images.githubusercontent.com/73488660/133363190-44d1b99c-5930-47a9-8b1f-92d9dc24dbfb.png)

### /password/reset
![screenshot-localhost_3000-2021 09 14-21_17_44](https://user-images.githubusercontent.com/73488660/133363278-58571c4d-78d6-40ee-bb34-fa80ab1a8ec1.png)

### /about/contact
- The `/about/store` path doesn't exist, so, if you click on submit button it will throw error.
![screenshot-localhost_3000-2021 09 14-21_19_11](https://user-images.githubusercontent.com/73488660/133363343-d46c70be-4d64-48de-aa2c-4b117782b869.png)

### /wiki/bibles
![screenshot-localhost_3000-2021 09 14-21_21_07](https://user-images.githubusercontent.com/73488660/133363617-1d50a669-651d-45ef-8c38-26dd00c8aac0.png)
